packages/use-wallet/src/wallets/defly-web.ts

```diff
@@ -0,0 +1,230 @@
+import { AVMProvider } from 'src/wallets/avm-web-provider'
+import { WalletId, type WalletConstructor } from 'src/wallets/types'
+import type AVMWebProviderSDK from '@agoralabs-sh/avm-web-provider'
+
+export const DEFLY_WEB_PROVIDER_ID = '95426e60-5f2e-49e9-b912-c488577be962'
+
+const ICON = `data:image/svg+xml;base64,${btoa(`
+<svg viewBox="0 0 1024 1024" xmlns="http://www.w3.org/2000/svg">
+  <rect width="1024" height="1024" />
+  <path fill="#FFFFFF" d="M779.9,684.4L512,230L244.1,684.4L512,529.5L779.9,684.4z" />
+  <path fill="#FFFFFF" d="M733.1,730L512,613.5L290.9,730L512,658L733.1,730z" />
+</svg>
+`)}`
+
+export class DeflyWebWallet extends AVMProvider {
+  constructor({
+    id,
+    store,
+    subscribe,
+    getAlgodClient,
+    metadata = {}
+  }: WalletConstructor<WalletId.DEFLY>) {
```

Comment:
I ran through these changes and replaced the Defly Web provider ID with the Kibisis one and it worked fine, but I I had to change this to `WalletId.DEFLY_WEB`.

I know Defly have their own UI for the mobile app, so I am unsure if this is supposed to be `WalletId.DEFLY`.